### DISCUSSION TITLE
What is OBRC?

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#999] [General]
Due to #994, what I see on https://orangec.readthedocs.io/en/latest/general/Obrc/ is only:

`OBRC generates SQLITE3 databases, with a file extension ".obr".`

Still sounds like ctags to me. Please correct me if I'm wrong.

**Comment by GitMensch** [comment]
See https://github.com/LADSoft/OrangeC/blob/master/doc/Tools.md for general information, or specific

https://github.com/LADSoft/OrangeC/blob/master/doc/general/Obrc.md
https://github.com/LADSoft/OrangeC/blob/master/doc/general/Occpr.md
https://github.com/LADSoft/OrangeC/blob/master/doc/general/Coff2IEEE.md

That said @LADSoft: it would be nice to have a short summary for the tool description that we have in https://github.com/LADSoft/OrangeC/blob/master/doc/Tools.md as part of `--help`.

```
d:\dev\orangec\bin>obrc --help
obrc (OrangeC) Version 6.73.1
Copyright (C) LADSoft 2006-2023

Usage: obrc [options] outputfile filelist

/V, --version  Show version and date
/!, --nologo   No logo
/?, --help     This text
@xxx           Read commands from file

Time: 23:11:36  Date: Oct 22 2023
d:\dev\orangec\bin>occpr --help
Copyright (C) LADSoft 2006-2023

Usage: occpr [options] [@response file] files

/1        - C1x mode                  /2        - c2x mode
/8        - c89 mode                  /9        - C99 mode
...


d:\dev\orangec\bin>Coff2ieee --help
off2ieee (OrangeC) Version 6.73.1
Copyright (C) LADSoft 2006-2023

Usage: Coff2ieee [options] <coff file>

/oxxx          Set output file name
/V, --version  Show version and date
/!, --nologo   No logo
/?, --help     This text


Time: 23:11:49  Date: Oct 22 2023
```

Note this also applies to common tools like olink!

Compare that with

```sh
$ objdump --help | head
Usage: objdump <option(s)> <file(s)>
 Display information from object <file(s)>.
```

A single sentence for each tool in its help is definitely useful... I'm moving this out to a separate issue.

**Comment by LADSoft** [comment]
ok sounds good...
